knox-homepage-ui/home/app/generalProxyInformation/general.proxy.information.component.ts

```diff
@@ -108,6 +116,7 @@ export class GeneralProxyInformationComponent implements OnInit {
         this['showAdminAPI'] = (profile['gpi_admin_api'] === 'true');
         this['showMetadataAPI'] = (profile['gpi_md_api'] === 'true');
         this['showTokens'] = (profile['gpi_tokens'] === 'true');
+        this['showWebShell'] = (profile['gpi_webshell'] === 'true');
```

Comment:
It will not work since you are missing the relevant profile declaration from `org.apache.knox.gateway.dto.HomePageProfile`